cli_tools/gce_ovf_export/exporter/instance_disks_exporter.go

```diff
@@ -145,7 +145,6 @@ func (ide *instanceDisksExporterImpl) addExportDisksSteps(w *daisy.Workflow, ins
 			"format":                     params.DiskExportFormat,
 			"export_instance_disk_image": "projects/compute-image-tools/global/images/family/debian-9-worker",
 			"export_instance_disk_size":  "200",
-			"export_instance_disk_type":  "pd-ssd",
```

Comment:
do we intentionally change the disk type?

Reply:
Good question -- I think I removed that since `pd-ssd` is the default value for that variable. That said, I think removing it here could cause confusion, so I added it back.